open-craft/edx-theme
# Hid course update dates in LMS and added message to add/edit form in CMS

Testing instructions:

1. Set up a devstack
2. Clone this repo and checkout this branch in your devstack `themes` directory
3. Add the following lines to your `edx-platform/cms/envs/private.py` and `edx-platform/lms/envs/private.py`:
```python
COMPREHENSIVE_THEME_DIRS = ["/edx/app/edxapp/themes/"]
THEME_NAME='cloudera'
USE_CUSTOM_THEME=True
ENABLE_COMPREHENSIVE_THEMING=True
```
4. Start Studio and the LMS (no fast mode)
5. Add the `cloudera` theme under Theme Sites in Studio and LMS administrative interfaces (`/admin`)
6. In Studio, go to the default course and click the `+ New Update` button. See the message "(hidden from OnDemand learners)" next to the date field

![captura de pantalla de 2017-02-19 20-48-48](https://cloud.githubusercontent.com/assets/560781/23110514/a11a535c-f6e7-11e6-9d41-334d1111e75d.png)

7. Post a couple of course updates
8. Go to the LMS and check there's no date above the udpates

![captura de pantalla de 2017-02-19 20-52-02](https://cloud.githubusercontent.com/assets/560781/23110520/aaf763e2-f6e7-11e6-95d4-3434e2009afa.png)
